.circleci/continue_config.yml

```diff
@@ -157,16 +155,29 @@ jobs:
                   cd services/QuillLMS
                   bundle exec rake db:schema:load
                   bundle exec rake db:migrate
+                  bundle exec rake local_data:e2e_seed
                 environment:
                   DATABASE_URL: "postgres://ubuntu@localhost:5432/quill_test_db"
+            - run:
+                name: vite build
+                command: |
+                  cd services/QuillLMS && bin/vite build
+            - run:
+                name: Start services with Foreman
+                command: cd services/QuillLMS && bundle exec foreman start -f Procfile.ci
+                background: true
             - run:
                 name: Run tests
                 command: |
                   cd services/QuillLMS
-                  TESTFILES=$(circleci tests glob "spec/system/*_spec.rb" | circleci tests split)
-                  bin/rspec ${TESTFILES}
+                  sudo npx playwright install-deps
+                  npx playwright install
+                  bundle exec rails runner "puts 'user id:' ; puts User.find_by_email('johnnyci@gmail.com').id"
+                  CI=true npx playwright test
+                environment:
+                  DATABASE_URL: "postgres://ubuntu@localhost:5432/quill_test_db"
             - store_artifacts:
-                path: /home/circleci/Empirical-Core/services/QuillLMS/tmp/screenshots
+                path: /home/circleci/Empirical-Core/services/QuillLMS/test-results
```

Comment:
I've found this quite useful when debugging in CircleCI, since we don't have a graphical connection to that environment.